### PR TITLE
[image][ios] Support accessibility props on iOS

### PR DESCRIPTION
**Pull request by alanjhughes**
# Why
Added `accessible` and `accessibilityLabel` props as discussed with @tsapeta 

# How
Added props to the native module

# Test Plan
Tested on a real device using Voiceover and on the simulator using the Accessibility Inspector.

### CODE REVIEW

File: packages/expo-image/ios/ImageView.swift [hunk 1]
```diff
@@ -40,6 +40,8 @@ public final class ImageView: ExpoView {
 
   var cachePolicy: ImageCachePolicy = .disk
 
+  var accessible: Bool = false
```
Comment (alanjhughes) [inline packages/expo-image/ios/ImageView.swift:43]: Could use a didSet here too?

File: packages/expo-image/ios/ImageModule.swift [hunk 2]
```diff
@@ -65,6 +65,14 @@ public final class ImageModule: Module {
         view.cachePolicy = cachePolicy ?? .disk
       }
 
+      Prop("accessible") { (view, accessible: Bool) in
```
Comment (alanjhughes) [inline packages/expo-image/ios/ImageModule.swift:68]: Does it need to be optional?

File: packages/expo-image/ios/ImageModule.swift [hunk 3]
```diff
@@ -65,6 +65,14 @@ public final class ImageModule: Module {
...
+        view.accessible = accessible
+      }
+
+      Prop("accessibilityLabel") { (view, label: String) in
```
Comment (alanjhughes) [inline packages/expo-image/ios/ImageModule.swift:72]: Same as above

File: packages/expo-image/ios/ImageModule.swift (hunk 2 above)
Comment (tsapeta) [reply packages/expo-image/ios/ImageModule.swift:68]: Yeah, better if it is.

File: packages/expo-image/ios/ImageModule.swift [hunk 4]
```diff
@@ -65,6 +65,14 @@ public final class ImageModule: Module {
...
+        view.accessibilityLabel = label
```
Comment (tsapeta) [inline packages/expo-image/ios/ImageModule.swift:73]: What about this? Then you don't need the `didSet` and separate member in `ImageView` 😉 
```suggestion
        view.sdImageView.accessibilityLabel = label
```

Comment (alanjhughes) [reply packages/expo-image/ios/ImageModule.swift:73]: 😄 I was going to just do that but I thought you mightn't want to access it directly like that

**Review by tsapeta — APPROVED** [review]
Thanks! 🙏

**Pull request merged**